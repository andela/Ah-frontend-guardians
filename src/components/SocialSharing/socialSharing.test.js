import React from 'react';
import { shallow } from 'enzyme';
import SocialShare from './socialSharing';

describe('Social Sharing tests', () => {
    const wrapper = shallow(<SocialShare/>);

    test('renders the component without crashing', () => {
        expect(wrapper).toMatchSnapshot()
    });
});
