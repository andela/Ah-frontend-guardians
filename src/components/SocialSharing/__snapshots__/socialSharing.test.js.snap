// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social Sharing tests renders the component without crashing 1`] = `ShallowWrapper {}`;
